framework/auth/views.py

```diff
@@ -199,10 +199,10 @@ def auth_logout(redirect_url=None):
     return resp
 
 
-def auth_email_logout(token, user):
+def auth_email_logout(token, user, redirect_url=None):
     """When a user is adding an email or merging an account, add the email to the user and log them out.
     """
-    redirect_url = web_url_for('auth_login')
+    redirect_url = cas.get_login_url(redirect_url)
```

Comment:
In every case I see, get_login_url is using a redirect_url that is set to none or pulled from a request.  How do you think I should handle it?  Set request_url=None in get_login_url?  Set request_url to '/' or whatever works in auth_email_logout?